packages/fiori/src/ViewSettingsDialog.hbs

```diff
@@ -0,0 +1,57 @@
+<ui5-dialog
+	aria-label="{{_dialogTitle}}"
+	@after-open={{_focusRecentlyUsedControl}}
```

Comment:
By default  both before-open and ui5-before-open events will be fired. But, Internally we use the "ui5" prefixed events, because there is a configuration option to keep them only (the use case is to prevent any event conflicts with other libraries etc. that might use the same event names) and then, the before-open and before-close won't be fired, but the ui5- ones will remain. 

So, here use @ui5-before-open and @ui5-before-close

Reply:
Done